baseline_collision.py

```diff
@@ -190,24 +190,22 @@ def make_step_prediction(choice, confidence, heatmap_img=None):
         choice = 1
     if choice == 'implausible':
         choice = 0
-    if heatmap_img:
+    if heatmap_img is not None:
         xy_list = np.argwhere(heatmap_img >= 200)
-        xy_list = [{"x": x, "y": y} for x, y in xy_list]
+        xy_list = [{"x": int(x), "y": int(y)} for x, y in xy_list]
     else:
         xy_list = None
     return {
             "rating": choice,
-            "score": choice,
-            "xy_list": xy_list
+            "score": confidence,
+            "violations_xy_list": xy_list
     }
 
 
-def run_collision_scene(json_path, controller, models, base_path="~/logs", num_steps=200, visualize=False, pred_horizon=6):
+def run_collision_scene(scene_data, controller, models, base_path="~/logs", num_steps=200, visualize=False, pred_horizon=6):
```

Comment:
For our eval pipeline, we just wanted to feed you the scene data as a dict, rather than the scene file's path.